### DISCUSSION TITLE
Is it possible to use nested weighted sublayers for incompatibilities?

### DISCUSSION THREAD

**Discussion by b-hasbmans** [nftchef/art-engine#131] [Help + Q&A]
Assume I have given folder structure

Eyes
|- Glitchy Eyes#10
|- Regular Eyes#10
|- Universal Eyes#10
Hats
|- Glitchy Hats#10
|- Hats Requiring No Eyes#10

Can I use the nested sub folders in incompatibilities? For example, could I exclude glitchy hats when glitchy eyes are present? How might I ensure hats requiring no eyes doesn't render any type of eye? Using the weighted nested sub folders in incompatibilities doesn't seem to do anything, I can share code tomorrow. Thanks!

**Comment by cryptofannatic** [comment]
@b-hasbmans did you figure this out?
